flink-connectors/flink-connector-hbase/src/main/java/org/apache/flink/connector/hbase/source/TableInputSplit.java

```diff
@@ -26,7 +26,7 @@
  * references to row below refer to the key of the row.
  */
 @Internal
-class TableInputSplit extends LocatableInputSplit {
+public class TableInputSplit extends LocatableInputSplit {
```

Comment:
We don't need to declare it `public`.

Reply:
@wuchong I have already removed this declaration.